### PR TITLE
User table creation migration

### PR DESCRIPTION
**Pull request by DniGaytan**
## What does this PR do?

1. Adds script that performs the migration of relevant user information to a single table called user.
2. Adds command to execute the script explained above.

## Related issues this PR closes or is related to

Closes none.

### CODE REVIEW

**Comment by DniGaytan** [comment]
@cbarrazaa1 ya puedes revisar este

File: scripts/user-table-migration.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+const admin = require('firebase-admin')
+
+// Check if environment variable for firebase
+// auth is available
+if(process.env.FIREBASE_SERVICE_ACCOUNT){
+    var serviceJson = Buffer.from(process.env.FIREBASE_SERVICE_ACCOUNT, 'base64').toString();
```
Comment (cbarrazaa1) [inline scripts/user-table-migration.js:6]: `const`

**Review by cbarrazaa1 — DISMISSED** [review]
lgtm, please run `yarn prettier --write ./scripts/user-table-migration.js` before merge

**Comment by DniGaytan** [comment]
@cbarrazaa1 ya esta actualizado

**Review by cbarrazaa1 — APPROVED** [review]

**Pull request merged**